OJP/OJP_Trips.xsd

```diff
@@ -878,6 +895,172 @@
 			<xs:element name="TripFare" type="TripFareResultStructure" minOccurs="0" maxOccurs="unbounded"/>
 		</xs:sequence>
 	</xs:complexType>
+	<xs:complexType name="UserPreferencesStructure">
+		<xs:annotation>
+			<xs:documentation>A collection of user preferences to allow the algorithm to narrow-down multimodal trip combinations to those that may be suitable for the user.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="LowPrice" default="average" minOccurs="0" maxOccurs="1">
+				<xs:annotation>
+					<xs:documentation>User's preference that the trip has a low price.</xs:documentation>
```

Comment:
[This extra text could be added to give more hints about the meaning and how it should be implemented]
If important, the trip planner should prioritize options (modes, lines, operators) known for low price, such as bus or coach.
If unimportant, also consider options which are usually more epensive, such as taxi or ridehailing, and also for longer distances.
E.g., take a taxi to the central station instead of a PT ride with transfers.